### PR TITLE
update bioconvert to v0.2.0

### PR DESCRIPTION
**Pull request by cokelaer**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by cokelaer** [comment]
@bioconda/core 

I got this error during the ci/circleci that I've never seen before:

    Undefined Jinja2 variables remain (['CONDA_BZIP2', 'CONDA_BZIP2']).  Please enable source 
    downloading and try again.  

It looks like this was reported in several issues (e.g. https://github.com/bioconda/bioconda-recipes/issues/7233) but I have not found any solutions for the time being. Could anyone help on this ? 

thanks in advance

File: recipes/bioconvert/meta.yaml [hunk 1]
```diff
@@ -34,20 +41,36 @@ requirements:
...
+    - pyexcel-xls
+    - xlrd
     - bamtools
     - bcftools
     - bedtools
     - bioawk
     - deeptools
-    - dsrc # [not osx]
+    - dsrc
+    - go ==1.10.3
```
Comment (corburn) [inline recipes/bioconvert/meta.yaml]: go is required and a specific version of it?

Comment (cokelaer) [reply recipes/bioconvert/meta.yaml]: go is required. We need 1.10.3 for now indeed.

File: recipes/bioconvert/meta.yaml [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+
 package:
   name: bioconvert
-  version: "0.0.10"
+  version: {{ version }}
 
 source:
-  url: https://pypi.python.org/packages/51/16/36baee3cf988d3f043e98341c43c05fd771f5dbeaef118ace23853e005d5/bioconvert-0.0.10.tar.gz
-  md5: a7073c2b80712e0884c6414eea4dee24
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (rvalieris) [inline recipes/bioconvert/meta.yaml]: you will need to define name as a variable like you did for version, for this to work.

**Pull request merged**